README.md

```diff
@@ -25,6 +25,12 @@
 | [RARE Technologies Student Incubator Program](https://rare-technologies.com/incubator/#details) | Yes |
 | [Open Summer of Code](http://2017.summerofcode.be/) | Yes** |
 | [Hyperledger Internship Program](https://wiki.hyperledger.org/internship/program_overview) | Yes |
+| [Snowplow](https://snowplowanalytics.com/blog/2015/04/09/announcing-our-summer-internship-program/) | Yes |
```

Comment:
The link you have given is to a blog post from 2015. Can you find a more recent one for us to know that it is conducted annually?

Reply:
Nevermind I found a recent one from their careers page. Replace your link with this

https://snowplowanalytics.com/company/careers/?gh_jid=1107068